AsyncDisplayKit.xcodeproj/project.pbxproj

```diff
@@ -1981,6 +2006,21 @@
 /* End PBXResourcesBuildPhase section */
 
 /* Begin PBXShellScriptBuildPhase section */
+		057DE97CCEF5219E324C2A17 /* 📦 Copy Pods Resources */ = {
+			isa = PBXShellScriptBuildPhase;
+			buildActionMask = 2147483647;
+			files = (
+			);
+			inputPaths = (
+			);
+			name = "📦 Copy Pods Resources";
```

Comment:
@maicki what version of Cocoapods is this?  The one that rewrote it to show [CP] is 1.0.1 which I think is the latest.